test/unit/core.ts

```diff
@@ -31,6 +30,17 @@ const patch = init(
   { experimental: { fragments: true } },
 );
 
```

Comment:
```suggestion
/** Shuffle an array using Durstenfeld's version of the Fisher–Yates shuffle. */
```

Reply:
this was added locally, rather than through the github interface to preserve the git commit message formatting